ocaml/quicktest/quicktest_cluster.ml

```diff
@@ -0,0 +1,92 @@
+
+open Quicktest_common
+(* provide test record type and make_test, start,
+ * debug, success, and failed test functions *)
+
+open Client
```

Comment:
It's strange to both open `Client` and to bind it as `C = Client`. You should only use one.